src/mina-signer/test-vectors/legacySignatures.ts

```diff
@@ -0,0 +1,206 @@
+import { DelegationJson, PaymentJson } from '../sign-legacy.js';
+
+// inputs and generated signatures from client_sdk/tests/test_signatures.js
+export { signatures, payments, delegations, strings, keypair };
+
+let keypair = {
+  privateKey: 'EKFKgDtU3rcuFTVSEpmpXSkukjmX4cKefYREi6Sdsk7E7wsT7KRw',
+  publicKey: 'B62qiy32p8kAKnny8ZFwoMhYpBppM1DWVCqAPBYNcXnsAHhnfAAuXgg',
+};
+
+let receiver = 'B62qrcFstkpqXww1EkSGrqMCwCNho86kuqBd4FrAAUsPxNKdiPzAUsy';
+
+let newDelegate = 'B62qkfHpLpELqpMK6ZvUTJ5wRqKDRF3UHyJ4Kv3FU79Sgs4qpBnx5RR';
+
+let payments: PaymentJson[] = [
+  {
+    body: { source: keypair.publicKey, receiver, amount: '42' },
+    common: {
+      fee: '3',
+      feePayer: keypair.publicKey,
+      nonce: '200',
+      validUntil: '10000',
+      memo: 'this is a memo',
+    },
+  },
+  {
+    body: { source: keypair.publicKey, receiver, amount: '2048' },
+    common: {
+      fee: '15',
+      feePayer: keypair.publicKey,
+      nonce: '212',
+      validUntil: '305',
+      memo: 'this is not a pipe',
+    },
+  },
+  {
+    body: { source: keypair.publicKey, receiver, amount: '109' },
+    common: {
+      fee: '2001',
+      feePayer: keypair.publicKey,
+      nonce: '3050',
+      validUntil: '9000',
+      memo: 'blessed be the geek',
+    },
+  },
+];
+
+let delegations: DelegationJson[] = [
+  {
+    body: { delegator: keypair.publicKey, newDelegate },
+    common: {
+      fee: '3',
+      feePayer: keypair.publicKey,
+      nonce: '10',
+      validUntil: '4000',
+      memo: 'more delegates, more fun',
+    },
+  },
+  {
+    body: { delegator: keypair.publicKey, newDelegate },
+    common: {
+      fee: '10',
+      feePayer: keypair.publicKey,
+      nonce: '1000',
+      validUntil: '8192',
+      memo: 'enough stake to kill a vampire',
```

Comment:
hahha. nice one.

Reply:
credit to Paul I believe for generating these test cases :D